templates/block/block--menu-block--news-topics.html.twig

```diff
@@ -44,13 +44,13 @@
 {% set attributes = attributes.setAttribute('id', heading_id) %}
 {% set attributes = attributes.addClass(classes) %}
 
-<div{{ attributes }}>
+<nav{{ attributes }} aria-labelledby="button-label">
   {# Label. If not displayed, we still provide it for screen readers. #}
   {% if not configuration.label_display %}
     {% set title_attributes = title_attributes.addClass('visually-hidden') %}
   {% endif %}
   {{ title_prefix }}
-  <button{{ title_attributes.setAttribute('class', menu_class) }}>
+  <button{{ title_attributes.setAttribute('class', menu_class) }} id="button-label">
```

Comment:
Attributes should be added to the attributes object and ids should use the clean_id function. Lines 43 and 44 in this file are examples of how to do that.

See: https://www.drupal.org/docs/8/theming-drupal-8/using-attributes-in-templates